cmd/storage-test.go

```diff
@@ -27,18 +27,18 @@ func NewStorageTestCommand() *cobra.Command {
 		Args:    cobra.NoArgs,
 		Run: func(cmd *cobra.Command, args []string) {
 			// Set up a connection to the server.
-			conn, err := grpc.Dial(addr, grpc.WithTransportCredentials(insecure.NewCredentials()))
+			client, err := grpc.New(addr)
```

Comment:
in `inventory-get.go` we do `inventory.New(addr)` and here we use `grpc.New()` what is the right way to make those similar usage... ? wdyt ?

Reply:
The `storage` package hasn't been implemented in the same way as `inventory`. Once this is done, they should have the same format. I just updated `storage-test` command with the `grpc.New()` call to leverage the common code.

Reply:
Ok. Looking forward to see what you can do for storage